### PR TITLE
Fix URL in git hosting stub article

### PR DESCRIPTION
**Pull request by thomaskuntzz**
https://github.com/freeCodeCamp/guides/commit/f7a4426ecd57b790a887b61f9f58a006034a0cb2#r27097759

### CODE REVIEW

File: src/pages/git/git-hosting/index.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: Git hosting
 ---
 ## Git hosting
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/git/tagging-in-git/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+This is a stub. <a href='https://github.com/freeCodeCamp/guides/blob/master/src/pages/git/git-hosting/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
```
Comment (systimotic) [inline src/pages/git/git-hosting/index.md]: These stubs are usually generated by the `bin/normaliseArticles.js` script. It uses the following template: `https://github.com/freecodecamp/guides/tree/master/src/pages/${filePath}/index.md`. The correct link here would be `https://github.com/freecodecamp/guides/tree/master/src/pages/git/git-hosting/index.md`. Any other link would be changed the next time that script runs.

Comment (thomaskuntzz) [reply src/pages/git/git-hosting/index.md]: Yes, still the wrong link... How should a stub be created then if there's a script?

Comment (systimotic) [reply src/pages/git/git-hosting/index.md]: @thomaskuntzz `bin/normaliseArticles.js` is the script that creates stubs, but it also does some other things that could mess with the repo right now 😅 We're working on cleaning it up (#7825 #752)

**Comment by thomaskuntzz** [comment]
Cool, thanks for merging!

However, I'm still unsure about how a stub should be created... You mentioned a script or something, what is the preferred way to created a stub?

**Comment by systimotic** [comment]
To follow up on https://github.com/freeCodeCamp/guides/pull/8014#discussion_r164729483, ideally we'd use that existing script, but it's a bit broken right now, so doing it manually is alright. I'm hoping we can fix that script, move the link normalization to a plugin, and maybe put the stub creation in a separate script. The crazy errors I'm getting on https://github.com/freeCodeCamp/guides/pull/739 aren't helping with this 😄

**Pull request merged**